std_msgs/Header header
string name
geometry_msgs/Pose pose
uint8[] mesh